appveyor.yml

```diff
@@ -44,6 +44,8 @@ build_script:
     dotnet pack src\Microsoft.Azure.WebJobs.Logging\WebJobs.Logging.csproj -o ..\..\buildoutput --no-build --version-suffix "-$env:APPVEYOR_BUILD_NUMBER"
 
     dotnet pack src\Microsoft.Azure.WebJobs.Logging.ApplicationInsights\WebJobs.Logging.ApplicationInsights.csproj -o ..\..\buildoutput --no-build --version-suffix "-$env:APPVEYOR_BUILD_NUMBER"
+
+    dotnet pack src\Microsoft.Azure.WebJobs.ServiceBus\EventHubs\WebJobs.Extensions.EventHubs.csproj -o ..\..\..\buildoutput --no-build --version-suffix "-$env:APPVEYOR_BUILD_NUMBER"
```

Comment:
Why not move all of the files into their own EventHub directory - it's a little confusing that the project is still under a ServiceBus directory. Is there are reason for it?

Reply:
yeah, should make it a lot easier to backport changes

Reply:
ok